libs/service-portal/driving-license/src/screens/DrivingLicense/DrivingLicense.tsx

```diff
@@ -49,11 +49,11 @@ function DrivingLicense(): JSX.Element {
         {(nationalRegistryUser?.age || 0) >= ELIGABLE_FOR_RENEWAL_AGE && (
           <Application
             heading={defineMessage({
-              id: 'service.portal:driving-license-renewal-heading',
+              id: 'sp.driving-license:driving-license-renewal-heading',
```

Comment:
Same here: Since ids are changing, let's de-duplicate the string.

```suggestion
              id: 'sp.driving-license:renewal-heading',
```